spec/services/submissions_inviter_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+describe SubmissionsInviter do
+  describe '#invite' do
+    let(:submission) { instance_double Submission }
+    let(:message_delivery) { instance_double ActionMailer::MessageDelivery }
```

Comment:
What if I want to ensure that this service uses the mailer I want it to use not just that it sends an email?